### PR TITLE
ruby, bundle, and yarn package update

### PR DESCRIPTION
**Pull request by ashpil**

### CODE REVIEW

**Review by hernanat — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -36,7 +36,7 @@ RUN bundle exec rake assets:precompile \
 	&& rm -rf node_modules tmp/cache vendor/assets lib/assets \
 	&& find ./app/assets -mindepth 1 -name config -prune -o -exec rm -rf {} +
 
-FROM ruby:2.7.1-alpine3.11
+FROM ruby:2.7.2-alpine3.11
```
Comment (ashpil) [inline Dockerfile:39]: Haven't tested this, though presumably it should just work?..

File: Gemfile [hunk 2]
```diff
@@ -1,7 +1,7 @@
 source "https://rubygems.org"
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby "2.7.1"
+ruby "2.7.2"
```
Comment (hernanat) [inline Gemfile:4]: did you run `bundle install` / `bundle update` after changing this?

**Review by hernanat — APPROVED** [review]
one question, looks good

File: Gemfile (hunk 2 above)
Comment (ashpil) [reply Gemfile:4]: yea

**Pull request merged**